.travis.yml

```diff
@@ -27,8 +26,7 @@ script:
   - export JDK6_HOME=$(jdk_switcher home openjdk6)
   - export JDK7_HOME=$(jdk_switcher home openjdk7)
   - export JDK8_HOME=$(jdk_switcher home oraclejdk8)
-  - export JDK9_HOME=/usr/lib/jvm/java-9-oracle
-  - test -d "${JDK9_HOME}" || export JDK9_HOME=$(jdk_switcher home oraclejdk8)
+  - export JDK9_HOME=$(jdk_switcher home oraclejdk9)
```

Comment:
`test -d "${JDK9_HOME}" ||` was there to make sure JDK9_HOME always points to some JDK home, so `toolchains.xml` contains path to some real JDK.
This prevents things like
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-toolchains-plugin:1.1:toolchain (default) on project postgresql: Misconfigured toolchains. Non-existing JDK home configuration at /home/travis/build/pgjdbc/pgjdbc -> [Help 1]
```

Could you add `test -d "${JDK9_HOME}" || ...` back?